pkgs/development/androidndk-pkgs/default.nix

```diff
@@ -22,6 +22,10 @@ let
       inherit (pkgs)
         stdenv
         runCommand wrapBintoolsWith wrapCCWith;
+
+      # For hardeningUnsupportedFlagsByTargetPlatform
+      llvmPackages = pkgs."llvmPackages_${llvmVersion}";
```

Comment:
Wouldn't it make more sense to directly pass `llvmPackages` as the argument to `makeNdkPkgs`?

Reply:
Yes, I think that's a better idea

Reply:
Done